app/test/test_model.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from test import *
+from lib.model import Bundle
+
+
+class TestModel(unittest.TestCase):
+
+    extractor = FixtureExtractor()
+
+    def test_bundle(self):
+        bundle = Bundle.from_extractor(self.extractor, 'bundles/' + vx_bundle_fqid)
+        self.assertEqual(vx_bundle, bundle)
```

Comment:
where are vx_bundle and vx_bundle_fqid coming from?

Reply:
It is an analysis bundle pulled from one of the recent integration test runs.